index.html

```diff
@@ -1899,5 +1899,12 @@ <h2>
     </section>
     <section id="idl-index"></section>
     <section id="conformance"></section>
+
+    <section id="contributors">
+      <h2>Contributors</h3>
+      <p>
+        Thank you to Rick Byers (Google), Brian Campbell (Ping Identity), Lee Campbell (Google), Nick Doty (CDT), Heather Flanagan (Spherical Cow Consulting), Ryan Galluzzo (NIST, ACD), Dominique Hazael-Massieux (W3C), Bjorn Hjelm (Yubico), Johann Hofmann (Google), Mike Jones (Self-Issued Consulting), Tobias Looker (MATTR), Matthew Miller (Cisco), Theresa O'Connor (Apple Inc.), Simone Onofri (W3C), Helen Qin (Google), Wendy Seltzer (Invited Expert), Manu Sporny (Digital Bazaar), Orie Steele (Transmute), Ted Thibodeau Jr (OpenLink Software), David Waite (Ping Identity), and Chris Wilson for your valuable feedback and contributions to this specification.
```

Comment:
Let's drop the affiliations... they will def get out of date.

Reply:
Yeah... this is exactly why these sections are not a great use of our time and limited time and resources. There's already a record of who was in the working group, their affiliations, and actual contributions via git commit history. Chasing down URLs, DID, or whatever for people doing their jobs does strike me as productive use of the group's or editor's time and pretty redundant.

Reply:
Ok, enjoy your time maintaining this (instead of doing work we actually need, like tests etc.). Please add all URLs and please add everyone who has contributed to the design.

Reply:
My comment was not directed to you, nor was it snark. It was a serious request to @MasterKale, who is an extremely talented engineer.